src/common/config/chartTypes/meter.js

```diff
@@ -0,0 +1,40 @@
+import colorPalettes from '../colorPalettes';
+
+const LabelColor =
+  getComputedStyle(document.documentElement).getPropertyValue(
+    '--kd-color-text-secondary'
+  ) || '#6d6d6d';
+
+export const type = 'meter';
+
+export const options = () => {
+  return {
+    radius: '80%',
+    circumference: 180,
+    rotation: 270,
+    plugins: {
+      datalabels: {
+        color: LabelColor,
+        font: function (context) {
+          var w = context.chart.width;
+          return {
+            size: w < 512 ? 10 : 12,
```

Comment:
Is it necessary to have variable font size here?

Reply:
I added this for responsive screen as we are displaying legend on meter area hence I made font 12px for large screen and 10px for small / mobile screen